pkg/client/testdata/default-plugins-via-selection.golden

```diff
@@ -24,6 +24,69 @@ metadata:
   namespace: sonobuoy
 ---
 apiVersion: v1
+data:
```

Comment:
Most of these changes to golden files are simply adding 1 of the default plugins to the YAML since these tests were just setting the PluginSelection field and not the `DynamicPlugins` field.